easybuild/easyconfigs/c/Catch2/Catch2-3.5.3-foss-2023a.eb

```diff
@@ -0,0 +1,34 @@
+##
+# Authors::   Guillermo Piccoli <guillermo.piccoli@hpcnow.com>
+##
+
+easyblock = 'CMakeMake'
+
+name = 'Catch2'
+version = '3.5.3'
+
+homepage = 'https://github.com/catchorg/Catch2'
+description = """A modern, C++-native, header-only,
+ test framework for unit-tests, TDD and BDD
+ - using C++11, C++14, C++17 and later
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Comment:
Hi Micket, I understood I should change the toolchain to GCCcore, and because I am using foss 2023a I should use GCCcore version 12.3.0 correct? 
How can I know in the future which packages should be at which level?